src/app/styles/tailwind.css

```
@@ -462,6 +462,24 @@ input.input {
   border: 1px solid var(--black);
 }
 
+.drag-item {
+  display: none;
+}
+
+.drag-over-item {
+  margin-left: 8px;
```

```suggestion
    transform: translateX(8px);
```

using transform doesn't shift the layout as much